sdk/src/Core/Amazon.Runtime/Credentials/AssumeRoleAWSCredentialsOptions.cs

```diff
@@ -85,5 +86,17 @@ public string MfaTokenCode
         /// operation.
         /// </summary>
         public string SourceIdentity { get; set; }
+
+        /// A list of session tags that you want to pass. Each session tag consists of a key name 
```

Comment:
Did you mean to include these comments in a `<summary>` tag? Otherwise they won't show up in intellisense (same comment applies to `TransitiveTagKeys`).

Reply:
@dscpinheiro Thanks for pointing out. I have fixed this is latest commit.